src/app/shared/components/icon-checkbox/custom-checkbox.component.ts

```diff
@@ -0,0 +1,19 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core'
+
+@Component({
+  selector: 'sofie-custom-checkbox',
+  templateUrl: './custom-checkbox.component.html',
+  styleUrls: ['./custom-checkbox.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CustomCheckboxComponent {
+  @Input() public label: string
+  @Input() public selected: boolean
+  @Input() public disabled: boolean
```

Comment:
These two variables can be prefixed with `is` to indicate that they are booleans.